src/components/icons/reset-chat.tsx

```diff
@@ -0,0 +1,28 @@
+import { SVGProps } from "react";
```

Comment:
Could you please consider including the word "**type**" in the **import**? This is because **SVGProps** are being used as a type.

The modified import statement should look like this:
import { type SVGProps } from "react";